packages/react-app/src/constants.js

```diff
@@ -9,6 +9,17 @@ export const BLOCKNATIVE_DAPPID = "0b58206a-f3c0-4701-a62f-73c7243e8c77";
 
 export const ALCHEMY_KEY = "oKxs-03sij-U_N0iOlrSsZFr29-IqbuF";
 
+/*
+Decrease the number of RPC calls. This value is being passed to the provider
+internal polling time (it'll look for new blocks after this time) and hooks
```

Comment:
also, we can update this 🙌

Reply:
Good catch! Fixed on 14822892abc4dad9b417af569715741e9979192d